### ISSUE TITLE
onboarding failed shortly after initiation

### ISSUE THREAD

**Issue by qwudssd** [awslabs/aws-saas-boost#388]
onboarding tenant process failed shortly after initiation with an error msg in the onboarding-validation log:
<img width="1318" alt="Screen Shot 2022-10-24 at 4 24 14 PM" src="https://user-images.githubusercontent.com/104920467/197622319-f5436a87-7243-4160-af1c-715323e4e7e2.png">

version branch: master

**Comment by muylucir** [comment]
I had same problem ECS service quota now change from reaource based quota to vcpu based quota until 15 Nov
. you can cuange ECS service quota mode using AWS CLI command below

```
aws ecs put-account-setting-default --name fargateVCPULimit --value enabled --region <your region>
```

after change vcpu based quota, you can provision new tenant.

**Comment by netapp-dhruv-tyagi** [comment]
Facing the same issue. Tried the above mentioned fix, but still seeing the same message:

`Quota will be exceeded for service Fargate On-Demand resource count. You are currently consuming 0.0, and Service Quota is 0.0.`

**Comment by qwudssd** [comment]
@muylucir thanks for your reply. but looks like im still having the same issue after running that command

**Comment by muylucir** [comment]
@qwudssd 
Is onboarding still a failure? Or do you only see the error message?

**Comment by qwudssd** [comment]
@muylucir yeah onboarding still failed

**Comment by qwudssd** [comment]
@muylucir the failure still occurred pretty shortly after the initiation with the same error. I'm trying to update my saas-boost with this fix https://github.com/awslabs/aws-saas-boost/pull/390, but there is an error in clouformation saying 
<img width="1018" alt="Screen Shot 2022-10-26 at 1 19 59 PM" src="https://user-images.githubusercontent.com/104920467/198093705-8fd2f53c-5c22-4136-b909-8a804da7f492.png">

have you encountered this error before?

**Comment by muylucir** [comment]
@qwudssd 
I think this is a new problem. The first question you asked me was already solved when you executed the AWS CLI command I told you, and when you applied fix #390. Because if the first problem hadn't been solved, the cloudformation stack wouldn't have even started.

If you are using the main branch now, it may not be stable. 
If the main purpose is not to test the latest version, I recommend using the stable version of tag v2.0.0 instead of the main branch.

**Comment by qwudssd** [comment]
@muylucir okay i see. thank you!
